### PR TITLE
Pd-134 new article true command administration initial config

### PR DESCRIPTION
**Pull request by clairewrtr**
This content already exists on the TC Admin index page. Propose moving it to the Getting Started section. Other articles changed to include minor image updates.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+tags:
+- tcinstall
+- tcadmin
+- tcconfig
+- tcresetpwd
+- scaletoptoolbar
+- coretoptoolbar
+---
+{{< toc >}}
+TrueCommand allows you to manage all your TrueNAS systems from one location. This multi-system management increases efficiency and simplifes operations. To get started with the initial configuration of TrueCommand, ensure that the following steps have been completed:
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:15]: Active voice, present tense
```suggestion
TrueCommand allows you to manage all your TrueNAS systems from one location. This multi-system management increases efficiency and simplifes operations. 
To get started with the initial configuration of TrueCommand, complete these steps:
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+* You have created the administrator account. See [Creating the Admin Account]({{< relref "/TrueCommand/TCGettingStarted/AdminAccounts.md" >}}) for more information.
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:17]: convert these to summary steps in an ordered list might be a better way to guide them through the objective, which is to configure using the Administration page:
```suggestion
1. Create the administrator account. See [Creating the Admin Account]({{< relref "/TrueCommand/TCGettingStarted/AdminAccounts.md" >}}) for more information.
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+* You have created the necessary user accounts. See [Creating User Accounts]({{< relref "/TrueCommand/TCGettingStarted/UserAccounts.md" >}}).
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:18]:
```suggestion
2. Create the necessary user accounts. See [Creating User Accounts]({{< relref "/TrueCommand/TCGettingStarted/UserAccounts.md" >}}).
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+* You familiar with the interface toolbars and icons. Review them here [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}).
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:19]: this is not necessarily a step, but we can make it one
```suggestion
3. Become familiar with the interface toolbars and icons. Review them here [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}).
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+* You have connected your first TrueNAS system. See [Connecting Your First TrueNAS System]({{< relref "/TrueCommand/TCGettingStarted/ConnectingTrueNAS.md" >}}) for more information.
+
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:21]: I think adding this general section provides a good introduction to the next set of sections and instructions.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+The **Administration** page, available to users with administrator permissions, displays additional system details and offers a variety of TrueCommand configuration options.
+Click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration** to access the **Administration** page. It is organized into **About**, **Certificates**, and **Configuration** panes.
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:23]: We are now moving away from using panes in favor of using widgets to describe these information cards.
```suggestion
Click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration** to access the **Administration** screen. It is organized into **About**, **Certificates**, and **Configuration** widgets (functional and information cards).
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+![TrueCommand231AdminSystemInfo](/images/TrueCommand/2.3.1/TrueCommand231AdminSystemInfo.png "Administration Configuration")
+
+## About
+
+The **About** pane contains the current TrueCommand system ID and version, contact information for iXsystems, and license details.
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:29]:
```suggestion
The **About** widget displays the current TrueCommand system ID and version, contact information for iXsystems, and license details. 
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+The **Administration** page, available to users with administrator permissions, displays additional system details and offers a variety of TrueCommand configuration options.
+Click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration** to access the **Administration** page. It is organized into **About**, **Certificates**, and **Configuration** panes.
+
+![TrueCommand231AdminSystemInfo](/images/TrueCommand/2.3.1/TrueCommand231AdminSystemInfo.png "Administration Configuration")
+
+## About
+
+The **About** pane contains the current TrueCommand system ID and version, contact information for iXsystems, and license details. 
+
+### Updating the License
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:31]: changing the heading level
```suggestion
#### Updating the License
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+
+You can expand TrueCommand to monitor more disks by upgrading or purchasing a license from iXsystems.
+Click **GET A LICENSE** to open a new browser tab to purchase a TrueCommand license.
+You can also contact iXsystems to upgrade the current license.
+
+After you upgrade or purchase a new license, you must upload it to TrueCommand.
+Click **Browse…** to open a file browser on your local system.
+Select the new license file to upload and click **UPLOAD LICENSE** to apply the new license to TrueCommand.
+
+## Certificates
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:41]: changing heading level and text
```suggestion
### Certificates Widget
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+
+The **Certificates** pane shows the certificates and Certificate Authorities (CAs) TrueCommand uses and has options to upload or import a certificate or CA.
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:43]: changing use of pane to widget
```suggestion
The **Certificates** widget shows the certificates and Certificate Authorities (CAs) TrueCommand uses and has options to upload or import a certificate or CA.
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 3 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:18]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 2 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:17]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 4 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:19]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 1 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:15]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:21]: Heading modified.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 6 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:23]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+* You have created the necessary user accounts. See [Creating User Accounts]({{< relref "/TrueCommand/TCGettingStarted/UserAccounts.md" >}}).
+* You familiar with the interface toolbars and icons. Review them here [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}).
+* You have connected your first TrueNAS system. See [Connecting Your First TrueNAS System]({{< relref "/TrueCommand/TCGettingStarted/ConnectingTrueNAS.md" >}}) for more information.
+
+The **Administration** page, available to users with administrator permissions, displays additional system details and offers a variety of TrueCommand configuration options.
+Click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration** to access the **Administration** page. It is organized into **About**, **Certificates**, and **Configuration** panes.
+
+![TrueCommand231AdminSystemInfo](/images/TrueCommand/2.3.1/TrueCommand231AdminSystemInfo.png "Administration Configuration")
+
+## About
```
[the remark this answers is not in the document]
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:27]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 7 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:29]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 8 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:31]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 9 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:41]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md (hunk 10 above)
Comment (clairewrtr) [reply content/TrueCommand/TCGettingStarted/InitialConfig.md:43]: Change made.

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+Clicking the **Configure** button displays the following fields:
+
+![TrueCommand231AdminAlertRecipientsConfigure](/images/TrueCommand/2.3.1/TrueCommand231AdminAlertRecipientsConfigure.png "Administration: Mailing List Configuration")
+
+| Field | Value |
+| ----- | ------------- |
+| Mailserver | Address for the SMTP server. |
+| Mailserver port | Port number the SMTP server listens to. |
+| Auth user | User name for plain authentication. |
+| Auth pass | Password for plain authentication. Leave blank for no-auth. |
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:137]:
```suggestion
| **Auth pass** | Password for plain authentication. Leave blank for no-auth. |
```

File: content/TrueCommand/TCGettingStarted/InitialConfig.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+| From | The email address of the sender. |
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/InitialConfig.md:138]:
```suggestion
| **From** | The email address of the sender. |
```

**Review by micjohnson777 — COMMENTED** [review]
Just a few things.

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**